nengo_gui/templates/page.html

```diff
@@ -94,6 +94,9 @@
                 <li id='Redo_last_button' role="presentation">
                     <a title='Redo last' class='glyphicon glyphicon-share-alt'> </a>
                 </li>
+                <li id='Forum_link' role="presentation" onclick="window.open('http://forum.nengo.ai');">
+                    forum
```

Comment:
It feels weird to have this lowercase plaintext right beside a bunch of icons... (I'll post a screenshot shortly). Also, the word "forum" isn't necessarily meaningful to our users... why would they go to the forum? Feels like we're giving them an answer to a question they don't have yet.

A few alternatives... not sure what my preference is:

1. Change the hotkey icon to [something else](http://getbootstrap.com/components/) (mabye glyphicon-info-sign? there's unfortunately no keyboard icon, though we could use a different iconset) and make the forum like the question mark icon.
2. Add the forum link to the Hotkeys modal instead, and maybe change the title to something like "Help" since that's what the icon suggests.
3. Change the text for "Need help?" or "Get help" or just "Help"